shared/actions/settings.js

```diff
@@ -479,6 +479,9 @@ const loadHasRandomPW = state =>
 // Mark that we are not randomPW anymore if we got a password change.
 const passwordChanged = () => SettingsGen.createLoadedHasRandomPw({randomPW: false})
 
+// $FlowIssue
+const stop = (_, action) => RPCTypes.ctlStopRpcPromise(action.payload.exitCode)
```

Comment:
I think flow complains because the param should be `{exitCode: action.payload.exitCode}`

Reply:
whoops, thanks again Danny!
This should wait till after code freeze though right?

Reply:
I was kind of hoping to get this in for the release. But if it's risky we can bump it to the next one.

Reply:
Seems a little risky, I'd say let it wait